### PR TITLE
Minor refactor to test code and handlers

### PR DESCRIPTION
**Pull request by Geod24**
Better reviewed commit-by-commit and with indentation diff off.

### CODE REVIEW

File: tests/Utils.ts [hunk 1]
```diff
@@ -134,3 +138,25 @@ export class TestAgora
...
+ * The test code allows the API server to be started and shut down.
+ */
+export class TestStoa extends Stoa
+{
+    constructor (agora_endpoint: URL, port: number | string)
+    {
+        super(":memory:", agora_endpoint, port, "127.0.0.1");
+    }
+
+    public stop () : Promise<void>
```
Comment (TrustHenry) [inline tests/Utils.ts]:
```suggestion
    public stop (): Promise<void>
```

File: tests/Recovery.test.ts [hunk 2]
```diff
@@ -63,24 +62,14 @@ class TestStoa extends Stoa
...
-    }
 }
 
 describe ('Test of Recovery', () =>
 {
     const agora_addr: URL = new URL('http://localhost:2820');
     const stoa_addr: URL = new URL('http://localhost:3837/');
     let agora_node: TestAgora;
-    let stoa_server : TestStoa;
+    let stoa_server : TestRecoveryStoa;
```
Comment (TrustHenry) [inline tests/Recovery.test.ts]:
```suggestion
    let stoa_server: TestRecoveryStoa;
```

**Comment by TrustHenry** [comment]
https://github.com/bpfkorea/stoa/issues/139 It will be resolved.

File: tests/Recovery.test.ts (hunk 2 above)
Comment (TrustHenry) [reply tests/Recovery.test.ts]: This is a previous style mistake. if you could fix it. Thanks

**Comment by Geod24** [comment]
Review addressed.

**Review by TrustHenry — APPROVED** [review]
LGTM

**Pull request merged**